### PR TITLE
Manage team on seller dashboard

### PR DESCRIPTION
**Pull request by Blackfry**
styling and fix dates on team brief view

### CODE REVIEW

File: src/bundles/BuyerDashboard/components/Dashboard/BuyerDashboard.css [hunk 1]
```diff
@@ -37,6 +32,7 @@
 
 .filters {
   margin: 1.5rem 0 0;
+  width: 74.48861%
```
Comment (matt-sm) [inline src/bundles/BuyerDashboard/components/Dashboard/BuyerDashboard.css]: thats a very precise percentage

Comment (Blackfry) [inline src/bundles/BuyerDashboard/components/Dashboard/BuyerDashboard.css]: That's precision! It was a number that was being used and I couldn't see that it was calculated anywhere, it worked though.

File: src/bundles/BuyerDashboard/components/DashboardBriefs/DashboardBriefs.css [hunk 2]
```diff
@@ -1,60 +1,184 @@
...
     }
 
-    .summary-item-field-heading-first {
-        vertical-align: bottom !important;
+    table.brief-table thead tr th.right-header {
+        width: 10%
     }
 }
 
+table.brief-table {
```
Comment (matt-sm) [inline src/bundles/BuyerDashboard/components/DashboardBriefs/DashboardBriefs.css]: is there some duplication in these table styles?

Comment (Blackfry) [inline src/bundles/BuyerDashboard/components/DashboardBriefs/DashboardBriefs.css]: It replaces styling from "content-tables" from a uikit style sheet. I don't think there's any doubling up within this file though.

File: src/bundles/BuyerDashboard/components/Dashboard/BuyerDashboard.css (hunk 1 above)
Comment (maxious) [inline src/bundles/BuyerDashboard/components/Dashboard/BuyerDashboard.css]: these exact percentages are rems or ems or something.

**Pull request merged**